src/subdomains/core/liquidity-management/entities/liquidity-management-order.entity.ts

```diff
@@ -11,10 +13,25 @@ export class LiquidityManagementOrder extends IEntity {
   @Column({ length: 256, nullable: false })
   status: LiquidityManagementOrderStatus;
 
+  @Column({ length: 256, nullable: true })
+  priceSource?: string;
```

Comment:
Ah, I think we don't need the price source. We can take `action.system` as the price source.